src/SourceMaps.ts

```diff
@@ -151,7 +152,10 @@ class SourceMapCache {
 			}
 		}
 		catch (e) {
-			throw new Error(`Failed to load source maps at [${this._sourceMapRoot}], check that 'sourceMapRoot' is set correctly.`);
+			if(e instanceof InlineSourceMapError) throw e;
```

Comment:
Always use braces :) 

```suggestion
			if(e instanceof InlineSourceMapError) { 
				throw e;
			}
```